content/docs/09-registries-and-packs/6-helm-charts.md

```diff
@@ -1,89 +1,82 @@
 ---
-title: "Helm Registry"
-metaTitle: "Spectro Helm Registry"
-metaDescription: "Description of Helm registries and charts and their usages within Spectro Cloud"
-icon: ""
+title: "Helm Registries"
+metaTitle: "Helm Registries"
+metaDescription: "Palette Dev Engine Helm Registries"
 hideToC: false
 fullWidth: false
 ---
 
 import Tabs from 'shared/components/ui/Tabs';
-import WarningBox from 'shared/components/WarningBox';
 import InfoBox from 'shared/components/InfoBox';
+import WarningBox from 'shared/components/WarningBox';
 import PointsOfInterest from 'shared/components/common/PointOfInterest';
 import Tooltip from "shared/components/ui/Tooltip";
 
 
-# Palette Helm Registry
-Helm Charts are a collection of Kubernetes resource files capable of deploying services ranging in varying complexities. Palette provides a few stable public Helm registries out of the box. Tenants can also add any public or private Helm registries to leverage charts. Palette makes deploying typical Kubernetes applications easier with native support for Helm charts. We also allow the addition of any public or private Helm chart registry to the system. Charts from these system registries or any additional ones added by tenant administrators are modeled into cluster profiles for cluster deployment.
-
-Palette mandates the use of Helm OCI version 3.7.0 and above for the best security experiences.
-
-#  Deploying Helm Charts
-Palette allows the deployment of Helm charts in two modes, one using the default Public Repo and another leveraging user's custom Helm registry.
-
-# Default System Helm Registry
-Palette provides users with a stable repo of Helm Charts accessible to all the tenants. This repository syncs all the Helm charts to the Palette system resources, enabling the tenants to model cluster profiles using out-of-the-box charts from the stable repo. It also improves ease of use of charts to the users.
-
-# Setting up Helm Registry with Tenant Scope
-
-Palette also provides an exclusive Helm Registry for tenants to add their private Helm charts. To create your private repo from the Palette console follow these steps:
-
-
-1. Login as a Tenant administrator, navigate to **Admin > Settings > Registries**.
 
+# Palette Helm Registry
+Helm Charts are a collection of Kubernetes resource files capable of deploying services of varying complexity. Palette provides some stable default Helm charts in its public Helm Chart Registry. 
 
-2. Click on **Add New Helm Registry** and provide the registry name, endpoint, and user credentials.
+Palette also leverages developers' custom Helm registries. You can add your own public or private Helm registries to Palette's Helm registry.
```

Comment:
```suggestion
Palette also supports creating custom Helm registries. You can add your own public or private Helm registries to Palette's Helm registry.
```
I think we can avoid specifying a specific person and keep it generic. There are other roles besides developers that use Palette. Many of them are system administrators or infrastructure analysts.